### DISCUSSION TITLE
Lockdown Winget

### DISCUSSION THREAD

**Discussion by ReneKierstein** [microsoft/winget-pkgs#90194] [Q&A]
Is it possibel to lockdown / restrict Winget using a GPO or Intune policy?
Would like to restrict users to eg. msstore only and the option to restict access to Winget completly

apparantly only available as GPO: https://techcommunity.microsoft.com/t5/windows-it-pro-blog/manage-windows-package-manager-with-group-policy/ba-p/2346322

Waiting for Intune settings

**Comment by Trenly** [comment]
Intune integration is being tracked by -
* https://github.com/microsoft/winget-cli/issues/215
